CHANGELOG.md

```diff
@@ -6,6 +6,12 @@
     Click to see more.
   </summary>
 
+(no new changes)
```

Comment:
can we add the regular headings here instead

Reply:
Done.